### ISSUE TITLE
Avoid from ... import *

### ISSUE THREAD

**Issue by tikk3r** [lmorabit/long_baseline_pipeline#74]
Ran into this because newer containers run a more recent Python 2 version than the "e75" container. This only affects `gains_toCS_h5parm.py` at the moment. For some reason this does not show up in Python 2.7.5 (e75...), but does become an issue on Python 2.7.15 (recent containers). In the case of `gains_toCS_h5parm.py`

https://github.com/lmorabit/lofar-vlbi/blob/38e3a87d9bcd8f16f550ebeb4d1e597a4d601623/bin/gains_toCS_h5parm.py#L17

overrides the `logging` variable, after which it is no longer the module but a Logger instance from LoSoTo (I believe). This leads to the following obscure crash:

```
Traceback (most recent call last):
  File "./gains_toCS_h5parm.py", line 238, in <module>
    format_stream = logging.Formatter("%(asctime)s\033[1m %(levelname)s:\033[0m %(message)s", "%Y-%m-%d %H:%M:%S")
AttributeError: 'Logger' object has no attribute 'Formatter'
```

From my tests it's solved by simply importing the required LoSoTo functions explicitely (from what I can see it's only `reorderAxes`, but maybe I've missed one):

```python
from losoto.lib_operations import reorderAxes
```

**Comment by lmorabit** [comment]
Ah, thanks for finding this.  I think reorderAxes is the only one.  I might also take the opportunity to update pyrap.tables to casacore.tables .

**Comment by lmorabit** [comment]
Tested and fixed with commit 82845d6b294149c6d2ef3167bdd9b68de6ce939d
